perceval/backends/_mps.py

```diff
@@ -284,7 +280,6 @@ def _transition_matrix_2_mode(self, u):
         u11, u12, u21, u22 = u[0, 0], u[0, 1], u[1, 0], u[1, 1]
         d = self._d
         big_u = np.zeros((d, d, d, d), dtype='complex_')  # matrix corresponding to action of BS on the 2 modes
-        # todo: find a way to vectorize and remove "for" loops
```

Comment:
Are we dropping this ?

Reply:
again a refinement - should be a task combined with putting this backend section into Quandelibc like we have for SLOS, for example